tools/cli/go-whisk-cli/commands/action.go

```diff
@@ -412,6 +412,11 @@ func getExec(artifact string, kind string, isDocker bool, mainEntry string) (*wh
         code, err = readFile(artifact)
         exec.Code = &code
 
+        if (ext == ".jar") {
```

Comment:
@dubeejw this fixes a bug in the CLI where --kind java doesn't encode the jar file.